pink/configuration.py

```diff
@@ -123,12 +126,16 @@ def check_limits(self, tol: float = 1e-6) -> None:
             if q_max[i] <= q_min[i] + tol:  # no limit
                 continue
             if self.q[i] < q_min[i] - tol or self.q[i] > q_max[i] + tol:
-                raise NotWithinConfigurationLimits(
-                    i,
-                    self.q[i],
-                    q_min[i],
-                    q_max[i],
-                )
+                if safe_break:
+                    raise NotWithinConfigurationLimits(
+                        i,
+                        self.q[i],
+                        q_min[i],
+                        q_max[i],
+                    )
+                else:
+                    print(f"Warning: Value {self.q[i]} at index {i} is out of limits: [{q_min[i]}, {q_max[i]}]")
```

Comment:
```suggestion
                    logging.warning(f"Warning: Value %f at index %d is out of limits: [%f, %f]", self.q[i], i, q_min[i], q_max[i])
```
We didn't have warnings so far in the project :blush: The benefit of using the `logging` package is that it lets users control what they want to do with warnings later in their application (e.g. disable them).